samples/go/csv/read.go

```diff
@@ -157,18 +157,34 @@ func readFieldsFromRow(row []string, headers []string, fieldOrder []int, kindMap
 	return fields
 }
 
+func primaryKeyFromFields(fields types.ValueSlice, fieldOrder, pkIndices []int) (types.ValueSlice, types.Value) {
```

Comment:
This is returning 2 results but the naming implies it is returning one thing - a primary key. Can you think of a better name? Also a comment would be good here.